model/vfs/file.go

```diff
@@ -24,21 +24,21 @@ type FileDoc struct {
 	// File revision
 	DocRev string `json:"_rev,omitempty"`
 	// File name
-	DocName string `json:"name"`
+	DocName string `json:"name,omitempty"`
 	// Parent directory identifier
 	DirID       string `json:"dir_id,omitempty"`
 	RestorePath string `json:"restore_path,omitempty"`
 
-	CreatedAt time.Time `json:"created_at"`
-	UpdatedAt time.Time `json:"updated_at"`
+	CreatedAt *time.Time `json:"created_at,omitempty"`
+	UpdatedAt *time.Time `json:"updated_at,omitempty"`
 
 	ByteSize   int64    `json:"size,string"` // Serialized in JSON as a string, because JS has some issues with big numbers
-	MD5Sum     []byte   `json:"md5sum"`
-	Mime       string   `json:"mime"`
-	Class      string   `json:"class"`
-	Executable bool     `json:"executable"`
+	MD5Sum     []byte   `json:"md5sum,omitempty"`
+	Mime       string   `json:"mime,omitempty"`
+	Class      string   `json:"class,omitempty"`
+	Executable *bool    `json:"executable,omitempty"`
 	Trashed    bool     `json:"trashed"`
```

Comment:
Why `Trashed` was kept as is (no `omitempty`)?

Reply:
I feel like `trashed` is more or less always relevant to be returned (at least as relevant as type, class, etc). Also, it would imply to turn `trashed` as a pointer, which has many more impact in the code.